gradle/libs.versions.toml

```diff
@@ -178,6 +179,7 @@ iceberg-core = { group = "org.apache.iceberg", name = "iceberg-core", version.re
 iceberg-api = { group = "org.apache.iceberg", name = "iceberg-api", version.ref = "iceberg" }
 iceberg-hive-metastore = { group = "org.apache.iceberg", name = "iceberg-hive-metastore", version.ref = "iceberg" }
 iceberg-gcp = { group = "org.apache.iceberg", name = "iceberg-gcp", version.ref = "iceberg" }
+iceberg-gcp-bundle = { group = "org.apache.iceberg", name = "iceberg-gcp-bundle", version.ref = "iceberg" }
```

Comment:
Have you added this to the file `LICENSE.bin`?

Reply:
ditto

Reply:
ok

Reply:
updated